pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -71,6 +71,20 @@ import (
 //     Responses:
 //       204: description: The alert rule was deleted successfully.
 
+// swagger:parameters RouteGetAlertRulesExport
+type AlertRulesExportParameters struct {
+	ExportQueryParams
+	// UID of folder from which export rules
+	// in:query
+	// required:false
+	FolderUID string `json:"folder_uid"`
```

Comment:
The JSON field for this in `ProvisionedAlertRule` is `folderUID` and in `AlertRuleGroup` is `folderUid`. Sadly different, but we should choose one of these instead of `folder_uid` IMO.